### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by Chupsy**

### CODE REVIEW

**Comment by Chupsy** [comment]
# [Codecov](https://codecov.io/gh/Chupsy/cats-cli/pull/6?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@bc9260f`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/Chupsy/cats-cli/pull/6?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##             master        #6   +/-   ##
==========================================
  Coverage          ?   100.00%           
==========================================
  Files             ?         1           
  Lines             ?         8           
  Branches          ?         1           
==========================================
  Hits              ?         8           
  Misses            ?         0           
  Partials          ?         0           
```

| Flag | Coverage Δ | |
|---|---|---|
| #unittests | `100.00% <0.00%> (?)` | |


------

[Continue to review full report at Codecov](https://codecov.io/gh/Chupsy/cats-cli/pull/6?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/Chupsy/cats-cli/pull/6?src=pr&el=footer). Last update [bc9260f...7955798](https://codecov.io/gh/Chupsy/cats-cli/pull/6?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: README.md [hunk 1]
```diff
@@ -26,6 +34,6 @@ yarn global add cats-cli
 
 ## Command Line
 
-Commands | Desciption | Options
---- | --- | ---
-```cats initialize``` | Create in the current directory a brand new project | _None_
\ No newline at end of file
+| Commands          | Desciption                                          | Options |
+| ----------------- | --------------------------------------------------- | ------- |
```
Comment (DORIAN-AQ) [inline README.md]: Why ? Auto table markdown already handle this. It adds complexity for the next edition ?

File: src/index.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+import program from 'commander'
+import './commands/init'
+clear()
 console.log(
-  chalk.red(figlet.textSync("CATS-CLI", { horizontalLayout: "full" }))
-);
-program.version("0.0.1").description("An example CLI for ordering pizza's");
+  chalk.red(figlet.textSync('CATS-CLI', { horizontalLayout: 'full' }))
+)
+program.version('0.0.1').description("An example CLI for ordering pizza's")
```
Comment (DORIAN-AQ) [inline src/index.ts]: Maybe we should the change the description here and get the version ?

File: README.md (hunk 1 above)
Comment (Chupsy) [reply README.md]: 🤷‍♂️ prettier does what prettier does ? :D

File: src/index.ts (hunk 2 above)
Comment (Chupsy) [reply src/index.ts]: actually haven't found a way to get version from package json since `/cats/src'. 'rootDir' is expected to contain all source files`

Comment (Chupsy) [reply src/index.ts]: changed the description though

Comment (DORIAN-AQ) [reply src/index.ts]: I'm not sure but this should be working : `const version = process.env.npm_package_version;`

File: README.md (hunk 1 above)
Comment (DORIAN-AQ) [reply README.md]: If the difficulty is handled by prettier, no problem then

**Pull request merged**